go.mod

```diff
@@ -1,3 +1,11 @@
 module github.com/medibloc/vc-sdk
 
 go 1.16
+
+require (
+	github.com/btcsuite/btcd v0.21.0-beta
+	github.com/hyperledger/aries-framework-go v0.1.6
+	github.com/stretchr/testify v1.7.0
+)
+
+replace github.com/hyperledger/aries-framework-go => github.com/medibloc/aries-framework-go v0.1.6
```

Comment:
For selective disclosure, I'm gonna fork and fix the [bls12-381](https://github.com/kilic/bls12-381) which is used in the [hyperledger/aries-framework-go](https://github.com/hyperledger/aries-framework-go). That's why I also forked the aries-framework-go. 

Now, I didn't make use the fixed version of aries-framework-go and bls12-381. Coming soon.